com.eduke32.EDuke32.yaml

```diff
@@ -0,0 +1,52 @@
+app-id: com.eduke32.EDuke32
+runtime: org.gnome.Platform
+runtime-version: '3.32'
+sdk: org.gnome.Sdk
+command: eduke32
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  - --socket=pulseaudio
+  - --device=dri
+  # For storing game files
+  - --filesystem=xdg-documents/EDuke32:create
+  # Unfortunately doesn't seem to respect XDG
+  - --persist=.config/eduke32
+add-extensions:
+  com.eduke32.EDuke32.Plugin:
+    versions: 1-3.32
+    directory: extensions
+    subdirectories: false
```

Comment:
Since you've also submitted `om.eduke32.EDuke32.Plugin.DukeNukem3DShareware`, you probably expect it to be mounted to `/app/extensions/DukeNukem3DShareware`. If so, `subdirectories` here must be `true`.

Reply:
EDuke32 doesn't seem to be able to handle subdirectories, unfortunately.

Reply:
So you want to mount the extension to `/app/extensions`, and it's the single extension? Then the extension ID should match exactly the extension point, without extra portion.